Dockerfile

```diff
@@ -0,0 +1,58 @@
+FROM pytorch/pytorch:2.5.1-cuda11.8-cudnn9-devel
+
+WORKDIR /workspace
+
+RUN apt-get update && \
+    apt-get install wget -y && \
+    apt-get install git -y
+
+# Install Miniconda
+RUN wget \
+    https://repo.anaconda.com/miniconda/Miniconda3-latest-Linux-x86_64.sh \
+    && mkdir /root/.conda \
+    && bash Miniconda3-latest-Linux-x86_64.sh -b \
+    && rm -f Miniconda3-latest-Linux-x86_64.sh
+
+RUN git clone https://github.com/nianticlabs/acezero --recursive
+
+RUN conda install ipykernel
+
+#Create ace0 env
+RUN cd acezero && \
+    conda env create -f environment.yml
+
+RUN echo "source activate ace0" > ~/.bashrc && \
+    conda run -n ace0 pip install ipykernel && \
+    conda install -n ace0 -c conda-forge libstdcxx-ng && \
+    /opt/conda/envs/ace0/bin/python -m ipykernel install --user --name=ace0
+
+RUN cd acezero/dsacstar && \
+    conda run -n ace0 python setup.py install
+
+#Install ace0 deps
+RUN apt-get install -y libnvidia-egl-wayland1
```

Comment:
Suggestion: Could you please combine as many apt-get calls as possible to a single RUN layer? See also https://docs.docker.com/build/building/best-practices/#apt-get

In general, it's good to have fewer RUN lines, and group instructions with `&& \` at the end. This allows the layer to not contain changes that would be undone in a consecutive command, and will result in smaller Docker images that are easier to carry around.